docs/sources/flow/tasks/send-profiles-to-pyroscope/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/send-profiles-to-pyroscope/
+canonical: https://grafana.com/docs/agent/latest/flow/send-profiles-to-pyroscope/
+description: Send profiles to Pyroscope 
+title: Send profiles Pyroscope 
+weight: 120
+---
+
+# Send profiles to Pyroscope
+
+Learn how to configure {{< param "PRODUCT_NAME" >}} to collect [Pyroscope Server][] profiles and forward them to a [Pyroscope Server][].
```

Comment:
```suggestion
Learn how to configure {{< param "PRODUCT_NAME" >}} to collect profiles and forward them to a [Pyroscope Server][].
```
I think that way there's less repetition. The profiles are in a standard pprof format AFAIK, so I don't think they need to be called "Pyroscope Server profiles".